shared/src/commonMain/kotlin/com/example/tasky/agenda/domain/AgendaRepository.kt

```diff
@@ -15,6 +19,7 @@ import com.example.tasky.database.model.ApiType
 import com.example.tasky.database.model.isDelete
 import com.example.tasky.login.domain.manager.SessionManager
 import dev.tmapps.konnection.Konnection
+import kotlinx.serialization.json.Json
 
 interface IAgendaRepository {
```

Comment:
Just your `IAgendaRepository` and your `AgendaRepository` that need to be in separate files to keep each to their own layer 🙂